.github/copilot-instructions.md

```
@@ -947,6 +947,151 @@ Learned patterns from previous problem-solving sessions:
+AI agent development questions     → AIAgentExpert
+Autonomous multi-step work         → claudette-auto
+Push through blockers fast         → Beast Mode
+Build Claude skills                → claude-skill
+Java migration to Azure            → AppModernization
+Multi-agent task coordination      → orchestration
+Research & solution planning       → Plan
+```
+
+### Decision Tree
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)